languages/__init__.py

```diff
@@ -12,17 +13,26 @@
 
 
 def execute(config, database, message, metadata: dict, data: io.BufferedReader):
-    lang = detect_language(metadata, data)
+    try:
+        lang = detect_language(metadata, data)
+        if not lang:
+            log.info("Could not detect language")
+            return
+        save_to_db(database, message, lang)
+    except LangDetectException:
+        log.info("Could not detect language")
 
-    save_to_db(database, message, lang)
 
 
 def detect_language(metadata, data):
     log.info('Detecting language of object')
     lang = metadata.get('lang', None) or metadata.get('language', None)
```

Comment:
An alternative to hard coding in `lang` and `language` could be to fetch the potential name from the configuration. Like creating a tag in the `custom` part of the plugin configuration, as in these examples:
* Faces plugin: [config](https://github.com/mabruras/sqapi-plugins/blob/master/faces/config.yml#L18) - [usage](https://github.com/mabruras/sqapi-plugins/blob/master/faces/__init__.py#L23-L24)
* Thumbnails plugin: [usage](https://github.com/mabruras/sqapi-plugins/blob/master/thumbnails/__init__.py#L21-L22)